btcutil/psbt/go.mod

```diff
@@ -3,20 +3,20 @@ module github.com/btcsuite/btcd/btcutil/psbt
 go 1.22
 
 require (
-	github.com/btcsuite/btcd v0.23.5-0.20231219003633-4c2ce6daed8f
-	github.com/btcsuite/btcd/btcec/v2 v2.1.3
-	github.com/btcsuite/btcd/btcutil v1.1.4
+	github.com/btcsuite/btcd v0.24.2
```

Comment:
nit: commit message is unnecessarily long (and mentions Golang version bump which doesn't happen). See https://github.com/btcsuite/btcd/blob/master/docs/code_contribution_guidelines.md#model-git-commit-messages.

Reply:
Yes you're right, I decided to revert the upgrade to go version, we should do that in a different PR across the entire repo imo.